.gitignore

```diff
@@ -21,6 +21,8 @@ xcuserdata
 *.moved-aside
 *.xcuserstate
 *.xcscmblueprint
+.DS_Store
```

Comment:
I'm cool with having these here but you should put them in your global gitignore.